### PR TITLE
New Kusto version (2019-09-07)

### PR DESCRIPTION
**Pull request by liatbezalel**
Generated and added tests by the new Kusto 2019-09-07 version.
This related to the following PRs:

https://github.com/Azure/azure-rest-api-specs/pull/7219
https://github.com/Azure/azure-rest-api-specs/pull/7300

### CODE REVIEW

File: sdk/kusto/Microsoft.Azure.Management.Kusto/src/Microsoft.Azure.Management.Kusto.csproj [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.Kusto</PackageId>
-    <Version>2.0.0</Version>
+    <Version>3.0.0</Version>
     <AssemblyName>Microsoft.Azure.Management.Kusto</AssemblyName>
     <PackageTags>management;kusto</PackageTags>
     <PackageReleaseNotes>
```
Comment (isra-fel) [inline sdk/kusto/Microsoft.Azure.Management.Kusto/src/Microsoft.Azure.Management.Kusto.csproj]: PackageReleaseNotes needs to be updated

**Review by isra-fel — CHANGES_REQUESTED** [review]
@liatbezalel please link your swagger spec review PR(s) here & update release notes, thanks

**Comment by isra-fel** [comment]
@liatbezalel ping on this

**Comment by liatbezalel** [comment]
> @liatbezalel please link your swagger spec review PR(s) here & update release notes, thanks

I updated the release notes and added the links to the relevant PRs in the description.
Thanks

**Review by isra-fel — APPROVED** [review]

**Pull request merged**